runtime/src/bank.rs

```diff
@@ -3904,14 +3904,17 @@ impl Bank {
     }
 
     pub fn force_flush_accounts_cache(&self) {
-        self.rc.accounts.accounts_db.force_flush_accounts_cache()
+        self.rc
+            .accounts
+            .accounts_db
+            .force_flush_accounts_cache(Some(self.slot()))
```

Comment:
nits: I think we can directly call `flush_accounts_cache({true/false}, max_clean_root);` from here. removing two very thin wrapper: `AccountsDB::force_flush_accounts_cache` and `AccountsDB::flush_accounts_cache_if_needed`